update.sh

```diff
@@ -17,7 +17,7 @@ echo "Updating certificates..."
 ./certs/update.sh
 
 echo "Fetching and building traefik $VERSION..."
-wget -O traefik https://github.com/emilevauge/traefik/releases/download/$VERSION/traefik
+wget -O traefik https://github.com/containous/traefik/releases/download/$VERSION/traefik
```

Comment:
Are you perhaps thinking of `curl`, whose [manual does feature `--fail`](https://curl.haxx.se/docs/manpage.html)?